docker/api/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gorghoa/php7
+
+WORKDIR /app
+
+RUN composer  create-project --no-scripts --no-dev api-platform/api-platform ./ v2.0.0-alpha.1 \
+    && rm -Rf src/AppBundle/Entity/*
+
+COPY ./start.sh ./start.sh
+RUN chmod +x ./start.sh
+
+RUN  composer require --dev api-platform/schema-generator
```

Comment:
Why installing dev dependencies if [you finally run project in prod](https://github.com/coopTilleuls/hm-admin/pull/20/files#diff-d629819f22b9bea4b73ee8fc7f571f00R9)?